trillian/integration/logenv.go

```diff
@@ -47,9 +47,9 @@ type CTLogEnv struct {
 // NewCTLogEnv creates a fresh DB, log server, and CT personality.
 // testID should be unique to each unittest package so as to allow parallel tests.
 // Created logIDs will be set to cfgs.
-func NewCTLogEnv(ctx context.Context, cfgs []*configpb.LogConfig, numSequencers int, testID string) (*CTLogEnv, error) {
+func NewCTLogEnv(ctx context.Context, cfgs []*configpb.LogConfig, numSequencers int) (*CTLogEnv, error) {
```

Comment:
Do you consider the "integration" package to have been implicitly internal?

Reply:
I think we can reasonably claim that.